src/sentry/static/sentry/app/components/modals/suggestProjectModal.tsx

```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {Observer} from 'mobx-react';
+import * as qs from 'query-string';
+
+import bgPattern from 'sentry-images/spot/mobile-hero.jpg';
+
+import {
+  addErrorMessage,
+  addLoadingMessage,
+  addSuccessMessage,
+} from 'app/actionCreators/indicator';
+import {ModalRenderProps} from 'app/actionCreators/modal';
+import {Client} from 'app/api';
+import Access from 'app/components/acl/access';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import List from 'app/components/list';
+import ListItem from 'app/components/list/listItem';
+import {t, tct} from 'app/locale';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import {trackAdvancedAnalyticsEvent} from 'app/utils/advancedAnalytics';
+import withApi from 'app/utils/withApi';
+import EmailField from 'app/views/settings/components/forms/emailField';
+import Form from 'app/views/settings/components/forms/form';
+import FormModel from 'app/views/settings/components/forms/model';
+
+type Props = ModalRenderProps & {
+  api: Client;
+  organization: Organization;
+  matchedUserAgentString: string;
+};
+
+type State = {
+  askTeammate: boolean;
+};
+
+class SuggestProjectModal extends React.Component<Props, State> {
+  state: State = {
+    askTeammate: false,
+  };
+  model = new FormModel();
```

Comment:
Why do you need to manually manage the form model?

Reply:
@EvanPurkhiser Because I didn't realize I could use the `extraButton` prop lol

Reply:
@EvanPurkhiser fixed